### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Cart Page

Live Preview: https://infallible-bell-91c11f.netlify.app/views/cart.html

### CODE REVIEW

File: views/cart.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
```
Comment (vanshitamoolchandani) [inline views/cart.html]: Change- the title

File: views/cart.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+                <a href="../views/cart.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">person</span>
+                        <span class="label-text label-text-primary">cart</span>
+                    </li>
+                </a>
+                <a href="../views/wishlist.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">queue</span>
+                        <span class="label-text label-text-primary">wishlisht</span>
```
Comment (vanshitamoolchandani) [inline views/cart.html]: Typo- Wishlist typo

File: views/cart.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+                </div>
+                <div class="ui-card-list-button gap flex flex-column">
+                    <div class="flex flex-row gap">
+                        <button class="btn-outline-primary btn pl-04 pr-04 pt-03">
+                            <span class="material-icons icon-menu">
+                                add
+                            </span>
+                        </button>
+    
+                        <input type="number" class="input input-qty" value="2" min="1">
```
Comment (vanshitamoolchandani) [inline views/cart.html]: Change- As you are using the type="number" the spinner provides click up and down arrows to select a number. So either you can remove that or remove + and - buttons.

You can refer this link:
https://css-tricks.com/snippets/css/turn-off-number-input-spinners/

File: views/cart.html [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+
+
+
+           
+          
+            
+
+
+
+
```
Comment (vanshitamoolchandani) [inline views/cart.html]: Suggestion- Remove extra space. If you want to diffrentiate between different components use comments.

**Review by kgajriaAgarwal — COMMENTED** [review]
All good.this can be merged.You can make changes suggested by other peer.

File: views/cart.html (hunk 1 above)
Comment (mehulsatardekar) [reply views/cart.html]: Done

File: views/cart.html (hunk 2 above)
Comment (mehulsatardekar) [reply views/cart.html]: done

File: views/cart.html (hunk 4 above)
Comment (mehulsatardekar) [reply views/cart.html]: done

File: views/cart.html (hunk 3 above)
Comment (mehulsatardekar) [reply views/cart.html]: done

Comment (mehulsatardekar) [reply views/cart.html]: @vanshitamoolchandani also Thanks for the reference link😁

**Comment by mehulsatardekar** [comment]
@vanshitamoolchandani @kgajriaAgarwal thanks you guys for suggesting changes. I have done all the changes.

File: views/cart.html [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+                </div>
+                <div class="flex flex-column gap-1">
+                    <span class="font-md-light">Deliver to:  <span class="label-text">John Doe, 560078</span></span>
+                    <span class="label-text">#1/4 , 100ft Ring Road, Jp Nagar - 4 Phase, Dollars Colony, Bangalore</span>
+
+                </div>
+
+            </div>
+
+           <div class="ui-card-default py">
```
Comment (pradipgharge) [inline views/cart.html:96]: use better class name instead of py

**Review by pradipgharge — APPROVED** [review]
Looks good to merge ! found no problems

**Review by kgajriaAgarwal — APPROVED** [review]

File: views/cart.html (hunk 5 above)
Comment (mehulsatardekar) [reply views/cart.html:96]: py describe padding to all btw these naming conventions, it is being used by a lot of CSS frameworks like (bootstrap,tailwindcss)

**Pull request merged**